From aaa8538723d91ad7a63e4b21fc36cc523be29b85 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:25:52 -0400
Subject: [redhat] kabi: Adding symbol wait_on_page_bit (mm/filemap.c)

Message-id: <9a84696104a48b9f074f7eb3dc780b530637be25.1410510773.git.poros@redhat.com>
Patchwork-id: 96061
O-Subject: [PATCH RHEL7.1 232/233] kabi: Adding symbol wait_on_page_bit (mm/filemap.c)
Bugzilla: 1071099
RH-Acked-by: Larry Woodman <lwoodman@redhat.com>

Bugzilla: 1071099
    https://bugzilla.redhat.com/show_bug.cgi?id=1071099

    Description:
    ============
    Not referenced directly, gets introduced due to inline helper
    function wait_on_page_writeback, which is used to wait for a page to complete witeback operation.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/wait_on_page_bit
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/wait_on_page_bit
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/wait_on_page_bit

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/wait_on_page_bit b/redhat/kabi/kabi-module/kabi_ppc64/wait_on_page_bit
new file mode 100644
index 0000000..48f7404
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/wait_on_page_bit
@@ -0,0 +1,2 @@
+#1-
+0x6824bdc4 wait_on_page_bit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/wait_on_page_bit b/redhat/kabi/kabi-module/kabi_s390x/wait_on_page_bit
new file mode 100644
index 0000000..940435d
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/wait_on_page_bit
@@ -0,0 +1,2 @@
+#1-
+0x5370dbf9 wait_on_page_bit vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/wait_on_page_bit b/redhat/kabi/kabi-module/kabi_x86_64/wait_on_page_bit
new file mode 100644
index 0000000..4fd63b7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/wait_on_page_bit
@@ -0,0 +1,2 @@
+#1-
+0x90c7c2cd wait_on_page_bit vmlinux EXPORT_SYMBOL
-- 
1.7.1